datasets/me/acp_peps/crawler.py

```diff
@@ -0,0 +1,212 @@
+
+
+def crawl_relative(context, person_entity, relatives):
+    for row in relatives:
+        name = row.pop("FUNKCIONER_IME")
+        surname = row.pop("FUNKCIONER_PREZIME")
+        relative_name = row.pop("IME_CLANA_PORODICE")
+        relative_surname = row.pop("PREZIME_CLANA_PORODICE")
+        # Skip if it's a self-reference
+        if name == relative_name and surname == relative_surname:
+            continue
+        maiden_name = row.pop("RODJENO_PREZIME_CLANA_PORODICE")
+        relationship = row.pop("SRODSTVO")
+        nationality = row.pop("DRZAVLJANSTVO")
+        city = row.pop("MESTO")
+        address = row.pop("BORAVISTE")
+
+        # Emit a relative
+        relative = context.make("Person")
+        relative.id = context.make_id(relative_name, relative_surname, maiden_name)
+        relative.add("address", address)
+        relative.add("nationality", nationality)
+        h.apply_name(
+            relative,
+            first_name=relative_name,
+            last_name=relative_surname,
+            maiden_name=maiden_name,
+        )
+        address_ent = h.make_address(
+            context,
+            full=address,
+            city=city,
+            lang="cnr",
+        )
+        h.copy_address(relative, address_ent)
+        context.emit(relative)
+
+        # Emit a relationship
+        rel = context.make("Family")
+        rel.id = context.make_id(person_entity.id, relationship, relative.id)
+        rel.add("relationship", relationship)
+        rel.add("person", person_entity.id)
+        rel.add("relative", relative.id)
+
+        context.emit(rel)
+        context.audit_data(row)
+
+
+def make_affiliation_entities(
+    context: Context, entity: Entity, function, row: dict, filing_date, report_id
+) -> List[Entity]:
+    """Creates Position and Occupancy provided that the Occupancy meets OpenSanctions criteria.
+    * A position's name include the title and the name of the legal entity
+    * All positions (and Occupancies, Persons) are assumed to be Montenegrin
+    """
+
+    organization = row.pop("ORGANIZACIJA")
+    start_date = row.pop("DATUM_POCETKA_OBAVLJANJA", None)
+    end_date = row.pop("DATUM_PRESTANKA_OBAVLJNJA")
+    context.audit_data(
+        row, ignore=["ORGANIZACIJA_IMENOVANJA", "ORGANIZACIJA_SAGLASNOSTI"]
+    )
+
+    position_name = f"{function}, {organization}"
+    entity.add("position", position_name)
+
+    position = h.make_position(
+        context,
+        position_name,
+        # topics=["gov.national"],  # for testing
+        country="ME",
+    )
+    apply_translit_full_name(
+        context, position, "cnr", position_name, TRANSLIT_OUTPUT, POSITION_PROMPT
+    )
+
+    categorisation = categorise(context, position, is_pep=True)
+    # Check categorisation topics match and fetch relatives
+    if "gov.national" in categorisation.topics:
+        relatives = fetch_csv(context, report_id, "csv_clanovi_porodice_")
+        crawl_relative(context, entity, relatives)
```

Comment:
we only want to emit the relatives if the PEP is still a PEP, so we need `if "gov.national" in categorisation.topics and entities`.

Maybe make_affiliation_entities can make a set of categorisation topics and return a tuple `(entities, topics)` then we call crawl_relative from crawl_person instead? Because just moving this down in this function will make us crawl the relative file for each function.